main/main.py

```diff
@@ -26,6 +27,7 @@
 
 parser = argparse.ArgumentParser(description='Open-source image labeling tool')
 parser.add_argument('-i', '--input_dir', default='input', type=str, help='Path to input directory')
+parser.add_argument('-n', '--n_frames', default='5', type=int, help='number of frames to track object for')
```

Comment:
I would increase this value, to `50` for example, otherwise, it looks like the tracker is not working. Also this flag needs to be added to the README.

Reply:
I will do this and also add another argument for selecting which tracking algorithm to use rather than having to hard code it. I'm looking into fixing the other issue you brought up now. I haven't run into this but I guess it's because I set it up working already.

Reply:
great! thank you so much!

Reply:
A potential solution is for me to fork the DaSiamRPN repo, make the necessary changes, and then link to my version of it instead of the original. Anything wrong with that approach?

Reply:
I don't have a better solution for now so let's do that.